### PR TITLE
Fixing issues with ignoredNamespaces on children

### PR DESCRIPTION
**Pull request by tomgco**
Previously it would ignore user defined ignores.

### CODE REVIEW

**Comment by jsdevel** [comment]
Can you add to `test/request-response-samples`?

File: lib/wsdl.js [hunk 1]
```diff
@@ -621,8 +621,10 @@ MessageElement.prototype._createLookupTypeObject = function (nsString, xmlns) {
  * @private
  */
 MessageElement.prototype._getNestedLookupTypeString = function (element) {
+  var ignoredNamespaces = this.options ? this.options.ignoredNamespaces
+      : this.ignoredNamespaces;
   var resolvedType = '^',
```
Comment (herom) [inline lib/wsdl.js]: Just an overall thought: wouldn't it be more sufficient if we merge/concat `this.options.ignoredNamespaces` with `this.ignoredNamespaces` on creation-time so we can call `this.ignoredNamespaces` in every case without having to re-assign and double-check it?

Comment (jsdevel) [inline lib/wsdl.js]: Seems you are right @herom.

**Comment by jsdevel** [comment]
Closing from lack of activity.

**Pull request closed without merging**